package.json

```diff
@@ -56,7 +56,7 @@
       "<rootDir>/testname"
     ],
     "snapshotSerializers": [
-      "<rootDir>/node_modules/pretty-format/build/plugins/convert_ansi"
+      "<rootDir>/node_modules/pretty-format/build/plugins/convertAnsi"
```

Comment:
```suggestion
      "<rootDir>/node_modules/pretty-format/build/plugins/ConvertAnsi"
```

Reply:
hah, it worked on mac 😅

Reply:
Seems like something must have changed in `jest-watcher` in between the update, `jest-watcher` has a lot of calls to `ansi-escapes`. I think that this is probably fine, but I'll take a deeper look

Reply:
This is the PR(https://github.com/facebook/jest/pull/7523) that introduces the snapshot change

Reply:
Would you say it's an expected change and it still works correctly?

Reply:
Technically yes, but we need to still mock something so that we can show it as expected in the snapshots. Similar to https://github.com/facebook/jest/pull/7523/files#diff-7291bbbfa0677c46d356c2910f279786R25

Reply:
I just pushed a commit that should address this.